### ISSUE TITLE
No theme found

### ISSUE THREAD

**Issue by RaitaroH** [OpenUserCSS/openusercss.org#64]
https://openusercss.org/theme/5a5503a243ab7f0b005ba4de
https://openusercss.org/theme/5a550a3343ab7f0b005ba4e6
My `Test` theme, that I deleted, still shows up in the New themes section, but it gives me an error if I try to delete it again:
```
Error: No theme found
Stack trace:
ExpectedError@https://openusercss.org/browser.js:1:3142625
AuthenticationError@https://openusercss.org/browser.js:1:3142939
_callee9/<@https://openusercss.org/browser.js:1:2951765
tryCatch@https://openusercss.org/browser.js:1:2139543
wrap/generator._invoke</<@https://openusercss.org/browser.js:1:2139151
defineIteratorMethods/</prototype[method]@https://openusercss.org/browser.js:1:2139825
step@https://openusercss.org/browser.js:1:2942492
step/<@https://openusercss.org/browser.js:1:2942661
run@https://openusercss.org/browser.js:1:1122212
notify/<@https://openusercss.org/browser.js:1:1122444
flush@https://openusercss.org/browser.js:1:1054065
MutationCallback*[220]</module.exports@https://openusercss.org/browser.js:1:1054471
[360]<@https://openusercss.org/browser.js:1:1121039
s@https://openusercss.org/browser.js:1:10749
s/<@https://openusercss.org/browser.js:1:10808
[478]<@https://openusercss.org/browser.js:1:1187249
s@https://openusercss.org/browser.js:1:10749
s/<@https://openusercss.org/browser.js:1:10808
[11]</<@https://openusercss.org/browser.js:1:163116
[11]<@https://openusercss.org/browser.js:1:162973
s@https://openusercss.org/browser.js:1:10749
s/<@https://openusercss.org/browser.js:1:10808
[709]</<@https://openusercss.org/browser.js:1:2923058
[709]<@https://openusercss.org/browser.js:1:2922338
s@https://openusercss.org/browser.js:1:10749
e@https://openusercss.org/browser.js:1:10923
@https://openusercss.org/browser.js:1:10514
@https://openusercss.org/browser.js:1:10470
@https://openusercss.org/browser.js:1:10177
```

There is also this:
```
Error: Cannot return null for non-nullable field Option.value.
Cannot return null for non-nullable field Option.value.
Cannot return null for non-nullable field Option.value.
Cannot return null for non-nullable field Option.value.
Cannot return null for non-nullable field Option.value.
Cannot return null for non-nullable field Option.value.
Cannot return null for non-nullable field Option.value.
Cannot return null for non-nullable field Option.value.
Cannot return null for non-nullable field Option.value.
Cannot return null for non-nullable field Option.value.
Stack trace:
ExpectedError@https://openusercss.org/browser.js:1:3142625
_callee5/<@https://openusercss.org/browser.js:1:2956321
tryCatch@https://openusercss.org/browser.js:1:2139543
wrap/generator._invoke</<@https://openusercss.org/browser.js:1:2139151
defineIteratorMethods/</prototype[method]@https://openusercss.org/browser.js:1:2139825
step@https://openusercss.org/browser.js:1:2952321
step/<@https://openusercss.org/browser.js:1:2952490
run@https://openusercss.org/browser.js:1:1122212
notify/<@https://openusercss.org/browser.js:1:1122444
flush@https://openusercss.org/browser.js:1:1054065
MutationCallback*[220]</module.exports@https://openusercss.org/browser.js:1:1054471
[360]<@https://openusercss.org/browser.js:1:1121039
s@https://openusercss.org/browser.js:1:10749
s/<@https://openusercss.org/browser.js:1:10808
[478]<@https://openusercss.org/browser.js:1:1187249
s@https://openusercss.org/browser.js:1:10749
s/<@https://openusercss.org/browser.js:1:10808
[11]</<@https://openusercss.org/browser.js:1:163116
[11]<@https://openusercss.org/browser.js:1:162973
s@https://openusercss.org/browser.js:1:10749
s/<@https://openusercss.org/browser.js:1:10808
[709]</<@https://openusercss.org/browser.js:1:2923058
[709]<@https://openusercss.org/browser.js:1:2922338
s@https://openusercss.org/browser.js:1:10749
e@https://openusercss.org/browser.js:1:10923
@https://openusercss.org/browser.js:1:10514
@https://openusercss.org/browser.js:1:10470
```

**Comment by DecentM** [comment]
When you click on a deleted theme, the client doesn't know if it still exists, so when it finds out that it doesn't, it removes it from localstorage.

Thanks for the second one, I'm tracking it in #65, because it only contains this issue, so I can close that one when it's fixed.

**Comment by DecentM** [comment]
Note to self: This needs checks when Loki is initialized.

**Comment by DecentM** [comment]
This should be fixed with 1.0.0. I'll close this issue, but if it still persists for you, please let me know!
